### PR TITLE
added install instruction for powershell users

### PR DESCRIPTION
**Pull request by jstacoder**
related to #583 install instructions for powershell users added to docs 

# Pull Request Check List

This is just a reminder about the most common mistakes. Please make sure that you tick all *appropriate* boxes.  But please read our [contribution guide](https://poetry.eustace.io/docs/contributing/) at least once, it will save you unnecessary review cycles!

- [] Added **tests** for changed code. << no code added
- [x] Updated **documentation** for changed code.

**Note**: If your Pull Request introduces a new feature or changes the current behavior, it should be based
on the `develop` branch. If it's a bug fix or only a documentation update, it should be based on the `master` branch.

If you have *any* questions to *any* of the points above, just **submit and ask**!  This checklist is here to *help* you, not to deter you from contributing!

### CODE REVIEW

**Comment by maggyero** [comment]
A very welcome addition. Thanks!

**Comment by maggyero** [comment]
A very welcome addition. Thanks @jstacoder!

I would add the `-UseBasicParsing` switch to avoid that `Invoke-WeRequest` parses the response body using the Internet Explorer DOM parser (cf. https://docs.microsoft.com/de-de/powershell/module/Microsoft.PowerShell.Utility/Invoke-WebRequest?view=powershell-3.0):
> -UseBasicParsing
> Indicates that the cmdlet uses the response object for HTML content without Document Object Model (DOM) parsing. This parameter is required when Internet Explorer is not installed on the computers, such as on a Server Core installation of a Windows Server operating system.

Without this parameter, machines where Internet Explorer is not installed raises the following exception (tested on a fresh Windows 10 Pro installation):

> The response content cannot be parsed because the Internet Explorer engine is not available, or Internet Explorer's first-launch configuration is not complete. Specify the UseBasicParsing parameter and try again.

Also, to access the `Content` property, I would use the grouping operator `()` instead of the subexpression operator `$()` because it is not necessary here.

So:

```
(Invoke-WebRequest -Uri https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py -UseBasicParsing).Content | python
```

File: docs/docs/index.md [hunk 1]
```diff
@@ -16,9 +16,14 @@ Poetry provides a custom installer that will install `poetry` isolated
 from the rest of your system by vendorizing its dependencies. This is the
 recommended way of installing `poetry`.
 
+### osx / linux / bashonwindows install instructions
 ```bash
 curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
 ```
+### windows powershell install instructions
+```powershell
+$(Invoke-WebRequest -Uri https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py).Content | python
```
Comment (maggyero) [inline docs/docs/index.md]:
```suggestion
(Invoke-WebRequest -Uri https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py -UseBasicParsing).Content | python
```

Comment (jstacoder) [reply docs/docs/index.md]: From the Microsoft docs:

`-UseBasicParsing`

> This parameter has been deprecated. Beginning with PowerShell 6.0.0, all Web requests use basic parsing only. 
> This parameter is included for backwards compatibility only and any use of it has no effect on the operation of the cmdlet.

Comment (jstacoder) [reply docs/docs/index.md]: Also the dollar sign is required so powershell knows to actually invoke the cmdlet to pass the result to the pipeline

Comment (maggyero) [reply docs/docs/index.md]: The problem is that Windows 10 and Windows Server 2019 ship with PowerShell 5.1 as the default version, not PowerShell 6, so the original command does not work for most users.

About the dollar sign, it is _not_ required, I have just retested and successfully installed Poetry without. The dollar sign is necessary for statements, but `Invoke-WebRequest -Uri {…}` is not a statement but an expression. See https://stackoverflow.com/questions/47274532/difference-between-and.

**Comment by maggyero** [comment]
Up!

**Comment by dlech** [comment]
I was just about to make the same pull request. It would have saved me quite a bit of time to have this information.

Also, I used `py -3` because `python` is not in my PATH on Windows.

**Comment by finswimmer** [comment]
Hello,

I'm not familiar with powershell. But I will have a look at the suggested command these days, to bring this PR forward.

Thanks a lot for your contributions!

**Pull request merged**